app/vulncode_app/data_dump.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2017 nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/vulnerablecode/
+# The VulnerableCode software is licensed under the Apache License version 2.0.
+# Data generated with VulnerableCode require an acknowledgment.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with VulnerableCode or any VulnerableCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with VulnerableCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  VulnerableCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  VulnerableCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/vulnerablecode/ for support and download.
+
+from vulncode_app.models import Vulnerability
+from vulncode_app.models import VulnerabilityReference
+from vulncode_app.models import ImpactedPackage
+from vulncode_app.models import ResolvedPackage
+from vulncode_app.models import Package
+from vulncode_app.models import PackageReference
+
+from scraper import debian, ubuntu
+
+
+def debian_data():
+    """
+    Scrape debian' security tracker.
+    """
+    json_data = debian.json_data()
+    extract_data = debian.extract_data(json_data)
+
+    return extract_data
+
+
+def debian_dump(extract_data):
+    """
+    Save data scraped from Debian' security tracker.
+    """
+    for data in extract_data:
```

Comment:
For example: `Vulnerability.objects.create(summary="Affected package xyz")`
-> `django.db.utils.IntegrityError: NOT NULL constraint failed: vulncode_app_vulnerability.cvss`
Since the cvss is not a nullable field, and since you do not set a value for it, your save() cannot work